### DISCUSSION TITLE
Release 2.5.1 (ESP8266/32 Mining fix)

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#607] [General]
## Hey there! This update should fix the latest problems while mining on ESP8266/32 boards.

Please re-upload the codes to your boards. 
Other than the corrected ESP codes, this release is the same as Release 2.5.
Have fun mining!

### ⚠️ NOTICE: .exe binaries may be detected as a virus by your antivirus software. This is a **false positive** caused by pyinstallers bootloader, read about it [here](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg). Duino-Coin is not a virus.

<hr /><em>This discussion was created from the release <a href='https://github.com/revoxhere/duino-coin/releases/tag/2.5.1'>Release 2.5.1 (ESP8266/32 Mining fix)</a>.</em>

**Comment by liubergarcia** [comment]
now it say i have to many workers ?how to fix i only have 2

**Reply by revoxhere** [reply]
Try now
